Documentation/contributor-guide/release.md

```diff
@@ -62,6 +62,7 @@ which don't need to be executed before releasing each version.
 4. Authenticate the image registry, refer to [Authentication methods](https://cloud.google.com/container-registry/docs/advanced-authentication).
    - `gcloud auth login`
    - `gcloud auth configure-docker`
+5. Ensure there is at least 500MB of free space on your Linux machine.
```

Comment:
Could you please follow https://github.com/etcd-io/etcd/pull/18346#discussion_r1685265865?

Reply:
Oops, did that one wrong... Thanks! Updated.